packages/devkit/devkit.1.20240429/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "igor@ahrefs.com"
+authors: "Ahrefs <github@ahrefs.com>"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/devkit"
+dev-repo: "git+https://github.com/ahrefs/devkit.git"
+bug-reports: "https://github.com/ahrefs/devkit/issues"
+synopsis: "Development kit - general purpose library"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@runtest"{with-test} "@doc"{with-doc}]
+]
+depends: [
+  "ocaml" {>= "4.05.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.13.0"}
```
Due to

[`failed: Unbound value StringLabels.starts_with`](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/fc5a99f1d19925c70c4559cbe5eb2b24d2d1a9bf/variant/compilers,4.12,devkit.1.20240429)